_posts/newsletters/2021-05-23-newsletter.md

```diff
@@ -0,0 +1,115 @@
+
+***You can still submit breakout sessions by Monday before the workshop!*** See more details [here](https://us-rse.org/virtual-workshop-2021/participate/).
+
+<a name="501c3"></a>
+# **US-RSE Open Collective 501(c)(3) created**
+
+US-RSE has formalized its membership in the Open Collective Foundation (OCF) fiscal host entity. OCF provides us an open platform for transparently managing the US-RSE long-term finances. Next steps will include a blog post that describes the sorts of things this could facilitate, and a breakout discussion at the 2021 US-RSE Virtual Workshop on possible fiscal priorities for the US-RSE.
+
+<a name="communitycall"></a>
+# **US-RSE Community Calls**
+
+We will have our next community call on June 10, 2021 at 12-1pm ET. Have a look nearer to the date in the slack channel [#communitycalls](https://usrse.slack.com/archives/CLBDQMJH5) to vote for a topic! You can find the videos from past community calls in our YouTube channel. Here is the one from [April](https://www.youtube.com/watch?v=9LwPO2bdHRE). We did not have a call in May since we will have the Second US-RSE Virtual Workshop on Monday and Thursday (May 24 and May 27). See above for more information.
+
+<a name="podcast"></a>
+# **Podcast**
+
+The Research Software Engineer Stories podcast has eight more episodes. Check them out below!
+* Daniel S. Katz - [Sustaining People](https://us-rse.org/rse-stories/2021/dan-katz/)
+* Paul Richmond - [Working with you rather than for you](https://us-rse.org/rse-stories/2021/paul-richmond/)
+* Ron Rahaman - [The Software Engineering Challenge](https://us-rse.org/rse-stories/2021/ron-rahaman/)
+* Mark Turner - [From Engineer to Manager](https://us-rse.org/rse-stories/2021/mark-turner/)
+* Daniel Nüst - [Are You Interested in Riddles?](https://us-rse.org/rse-stories/2021/daniel-nuest/)
+* Frank Loeffler - [It's a process](https://us-rse.org/rse-stories/2021/frank-loeffler/)
+* Edd Salkield - [Dreaming Spires](https://us-rse.org/rse-stories/2021/edd-salkield/)
+* Mark Abraham - [Career Structures for RSEs](https://us-rse.org/rse-stories/2021/mark-abraham/)
+
+<a name="events"></a>
+# **Interesting Events**
+
+Some other events coming up that may be of interest to RSEs include:
+* Internet2 has a virtual series of events with some of interest for the RSE community. You can see the overview of upcoming events [here](https://internet2.edu/upcoming-events/i2-online/). July 1 will be a [panel discussion](https://internet2.edu/upcoming-events/i2-online/#july) moderated by Sandra Gesing on sustainability of science gateways, for example. [Registration](https://service5.internet2.edu/reg/events/i2cv-0701/registrations) is required and free of charge.
+* [SciPy 2021](https://www.scipy2021.scipy.org/register) will be held July 12 -18 2021. Registration is open, and you can apply to receive [financial support](https://www.scipy2021.scipy.org/financial-aid) through May 31.
```

Comment:
```suggestion
* [SciPy 2021](https://www.scipy2021.scipy.org/register) will be held July 12-18 2021. Registration is open, and you can apply to receive [financial support](https://www.scipy2021.scipy.org/financial-aid) through May 31.
```
or add spaces but also to the one below.

Let's not get into if to use a hyphen or en-dash. :smile: